modular_robot_simulation/revolve2/modular_robot_simulation/build_multi_body_systems/concrete_builder/_get_builder.py

```diff
@@ -0,0 +1,29 @@
+from revolve2.modular_robot.body import Module
+from revolve2.modular_robot.body.common import ActiveHinge, Brick
+from revolve2.modular_robot.body.v1 import CoreV1
+from revolve2.modular_robot.body.v2 import CoreV2
+
+from .._builder import Builder
+from ._active_hinge_builder import ActiveHingeBuilder
+from ._brick_builder import BrickBuilder
+from ._core_v1_builder import CoreV1Builder
+from ._core_v2_builder import CoreV2Builder
+
+
+def get_builder(module: Module) -> Builder:
+    """
+    Get and initialize the corresponding Builder for module.
+
+    :param module: The target Module.
+    :return: The initialized Builder.
+    :raises KeyError: If no Builder available for the Module type.
+    """
+    try:
+        return {
+            CoreV1: CoreV1Builder(),
```

Comment:
Same as with the ci_group modular robots, you can do this mapping but now you are creating the objects, even though you will not use all but one of them